tests/quick_test.py

```
@@ -0,0 +1,77 @@
+print("✅ Agent 初始化测试通过")
+
+# 测试 2: 工具注册
+print("\n2. 测试工具注册...")
+registry = get_tool_registry()
+tool_list = registry.get_tool_list()
+tool_names = [tool["function"]["name"] for tool in tool_list]
+assert "read_file" in tool_names
+assert "list_files" in tool_names
+assert "edit_file" in tool_names
+assert "run_bash" in tool_names
+print(f"✅ 工具注册测试通过，工具: {tool_names}")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)